src/tools/annotation/FreehandMouseTool.js

```diff
@@ -810,6 +813,13 @@ export default class FreehandMouseTool extends BaseAnnotationTool {
     external.cornerstone.updateImage(eventData.element);
   }
 
+  /**
+   *
+   */
+  _measurementModifyCallback(evt) {
```

Comment:
Adding a callback and adding an internal `MEASUREMENT_MODIFIED` event listener throughout this code would only make sense if the callback does something. If you intend to add some functionality to this callback. I'm happy to wait and re-review.

However, if its just being left here as placeholder I see no benefit in adding the callback and associated `addEventListener`/`removeEventListener` commands at this time.

Reply:
I think so.

So I will delete this callback and EventListeners. thanks for review.